test/runtests.jl

```diff
@@ -14,13 +14,4 @@ include("test_setmacro.jl")
 include("test_setindex.jl")
 include("test_functionlenses.jl")
 
-using Documenter: doctest
```

Comment:
I think running doctests here just causes more headaches than it adds values. So let's remove them. They are run in the docs build anyway.